3p/iframe-messaging-client.js

```diff
@@ -57,6 +57,15 @@ export class IframeMessagingClient {
   }
 
   /**
+   *  Send a postMessage to Host Window
+   *  @param {object} message The message to send.
+   *  @private
```

Comment:
Will Closure Compiler allow this? I don't think you can touch a super class's private methods, only protected or public.